Makefile

```diff
@@ -38,6 +39,12 @@ deploy: ## deploy a container to Nomad
 show-coverage:
 	xdg-open coverage/lcov-report/index.html
 
+typecheck: ## run typecheck, -w flag for watchmode
```

Comment:
if you like the above suggestion on `-w` instructions please update this one to be consistent ;)